src/transformers/models/whisper/generation_whisper.py

```diff
@@ -1584,7 +1620,11 @@ def _retrieve_segment(
     ):
         # find the predicted "end of segment" predictions of Whisper
         # "end of segment" predictions occur whenever Whisper predicts a timestamp token
-        timestamp_tokens: torch.Tensor = seek_sequence.ge(timestamp_begin)
+        if timestamp_begin is not None:
```

Comment:
Changes should be able to be reverted after setting `timestamp_begin = self.config.vocab_size + 1`, since then no element in `seek_sequence` can ever be `ge` than `timestamp_begin`